README.md

```diff
@@ -89,7 +89,21 @@ qmgrConnectionParams:
 # How long to wait until metrics are published by queue manager (milliseconds).
 # Value must be at least 10000 (periodicity with which metrics are published by MQ).
   connTimeout: 12000
-
+# Use TLS connection to queue manager?
```

Comment:
I suggest to note that if "useTLS" equals "false", all connection parameters below will be ignored.